### PR TITLE
feat: added docs for course live app

### PR DESCRIPTION
**Pull request by AhtishamShahid**
Ticket: https://openedx.atlassian.net/browse/TNL-9289

Added Documentation for Course Live App documentation.

- [ ] Product review:

ScreenShot: 

11 6  Course Live — Building and Running an edX Course documentation

### CODE REVIEW

File: en_us/shared/course_assets/course_live.rst [hunk 1]
```diff
@@ -0,0 +1,84 @@
+.. _Course Live:
+
+###########################
+Course Live
+###########################
+
+You can add live video call providers like zoom-lti-pro to your course through the course authoring app.
```
Comment (asadazam93) [inline en_us/shared/course_assets/course_live.rst]: video conferencing

Comment (asadazam93) [reply en_us/shared/course_assets/course_live.rst]: via the course authoring

File: en_us/shared/course_assets/course_live.rst [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+Initially, PII sharing is not enabled for the courses. edX's Live configuration interface in
```
Comment (asadazam93) [inline en_us/shared/course_assets/course_live.rst]: By default PII sharing is not enable for a course.

File: en_us/shared/course_assets/course_live.rst [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    This configuration will
+    require sharing usernames and emails of learners and the course team with {provider}.
+    To access the LTI configuration for {provider}, please request your edX project coordinator
+    to get PII sharing enabled for this course.
+
+**************************
+Allow PII sharing
+**************************
+
+To allow PII sharing for the course user should have access to this model https://example.com/admin/lti_consumer/courseallowpiisharinginltiflag
```
Comment (asadazam93) [inline en_us/shared/course_assets/course_live.rst]: for a course

File: en_us/shared/course_assets/course_live.rst [hunk 4]
```diff
@@ -0,0 +1,84 @@
+.. _Course Live:
+
+###########################
+Course Live
+###########################
+
+You can add live video conferencing providers like zoom-lti-pro to your course via the course authoring app.
+
+By default PII sharing is not enable for a course. edX's Live configuration interface in
```
Comment (asadazam93) [inline en_us/shared/course_assets/course_live.rst]: enabled

**Review by asadazam93 — APPROVED** [review]
Just a nit

**Review by awais-ansari — APPROVED** [review]

**Review by asadazam93 — APPROVED** [review]

**Pull request merged**